test_runner/fixtures/neon_fixtures.py

```diff
@@ -1294,7 +1294,7 @@ def __init__(self, env: NeonEnv):
     def raw_cli(
         self,
         arguments: List[str],
-        extra_env_vars: Optional[Dict[str, str]] = None,
+        extra_env_vars: Dict[str, str] = {},
```

Comment:
https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/ 

It's a potential bug when you put as the default argument a mutable object.

Reply:
Wow, that's a big foot-gun! I changed it back. I wish there was a way to pass a Dict by-value instead..

Reply:
usually, it does in a such way

```python
def f(d: Optional[Dict[str, str]] = None): 
  d = d or {}
```

the same situation with `list`